doc/write_your_own/connections/interactions_2.ipynb

```diff
@@ -120,11 +120,20 @@
     "        IPython.display.display(formatting[i])    \n",
     "confirm_button.on_click(custom_options)\n",
     "\n",
-    "# container is a box with all the selectors I want to display in it.\n",
+    "# # container is a box with all the selectors I want to display in it.\n",
```

Comment:
Also, given that the second code cell in this notebook is unavoidably huge, perhaps it would be worth noting in a text cell at the top why it can't be split down.